src/__tests__/components/studies/app-design/AppDesignTest.test.tsx

```diff
@@ -1,367 +1,369 @@
-
-  const IRBApprovalNumber = getById(
-    appDesign as HTMLElement,
-    'IRB-approval-input'
-  )
-  userEvent.type(IRBApprovalNumber!, TESTING_TEXT)
-  userEvent.click(container)
-  studyObject.irbProtocolId = TESTING_TEXT
-  expect(onUpdate).lastCalledWith(studyObject)
-})
-
-test('should study summary section behave correctly', () => {
-  const officialStudyName = getById(
-    appDesign as HTMLElement,
-    'study-name-input'
-  )
-  userEvent.type(officialStudyName!, TESTING_TEXT)
-  userEvent.click(container)
-  studyObject.name = TESTING_TEXT
-  expect(onUpdate).lastCalledWith(studyObject)
-
-  renderAppDesignComponent()
-  const studySummaryBody = getById(appDesign as HTMLElement, 'study-body-text')
-  userEvent.type(studySummaryBody!, TESTING_TEXT)
-  userEvent.click(container)
-  studyObject.details = TESTING_TEXT
-  expect(onUpdate).lastCalledWith(studyObject)
-})
-
-test('should error text appear when emails are badly formatted', () => {
-  const ethicsBoardEmail = getById(
-    appDesign as HTMLElement,
-    'ethics-email-input'
-  )
-  userEvent.type(ethicsBoardEmail!, TESTIING_TEXT_BAD_EMAIL)
-  userEvent.click(container)
-  expect(
-    getById(appDesign as HTMLElement, 'ethics-bad-email-text')
-  ).not.toBeNull()
-  userEvent.clear(ethicsBoardEmail!)
-  userEvent.click(container)
-  expect(getById(appDesign as HTMLElement, 'ethics-bad-email-text')).toBeNull()
-
-  const contactLeadEmail = getById(
-    appDesign as HTMLElement,
-    'contact-email-input'
-  )
-  userEvent.type(contactLeadEmail!, TESTIING_TEXT_BAD_EMAIL)
-  userEvent.click(container)
-  expect(
-    getById(appDesign as HTMLElement, 'general-contact-bad-email-text')
-  ).not.toBeNull()
-  userEvent.clear(contactLeadEmail!)
-  userEvent.click(container)
-  expect(
-    getById(appDesign as HTMLElement, 'general-contact-bad-email-text')
-  ).toBeNull()
-})
-
-test('should error text appear when phone numbers are badly formatted', () => {
-  const ethicsBoardPhoneNumber = getById(
-    appDesign as HTMLElement,
-    'ethics-phone-number-input'
-  )
-  userEvent.type(ethicsBoardPhoneNumber!, TESTING_TEXT_BAD_PHONE_NUMBER)
-  userEvent.click(container)
-  expect(
-    getById(appDesign as HTMLElement, 'ethics-phone-bad-text')
-  ).not.toBeNull()
-  userEvent.clear(ethicsBoardPhoneNumber!)
-  userEvent.click(container)
-  expect(getById(appDesign as HTMLElement, 'ethics-phone-bad-text')).toBeNull()
-
-  const contactLeadPhoneNumber = getById(
-    appDesign as HTMLElement,
-    'phone-number-contact-input'
-  )
-  userEvent.type(contactLeadPhoneNumber!, TESTING_TEXT_BAD_PHONE_NUMBER)
-  userEvent.click(container)
-  expect(
-    getById(appDesign as HTMLElement, 'general-contact-bad-phone-text')
-  ).not.toBeNull()
-  userEvent.clear(contactLeadPhoneNumber!)
-  userEvent.click(container)
-  expect(
-    getById(appDesign as HTMLElement, 'general-contact-bad-phone-text')
-  ).toBeNull()
-})
+// import {BrowserRouter} from 'react-router-dom'
```

Comment:
Why `//` rather than `/*`?

Reply:
I highlighted everything and hit command + / , which uses `//`. Should i change it?

Reply:
I suppose if there is an quick way to uncomment it - it's fine. In general // is for single line comments and /*. */ for blocks